dbtmetabase/__init__.py

```diff
@@ -213,6 +241,29 @@ def main(args: List = None):
         default=False,
         help="Append tags to Table descriptions in Metabase",
     )
+    parser.add_argument(
```

Comment:
Arguments that only apply to exposures should go in a subparser. Here's an example https://www.reddit.com/r/learnpython/comments/7xj2f2/correct_way_to_make_nested_argparse/du8qcs2

Reply:
> Arguments that only apply to exposures should go in a subparser. Here's an example https://www.reddit.com/r/learnpython/comments/7xj2f2/correct_way_to_make_nested_argparse/du8qcs2

I opted for arg groups because when I set up subparsers, it split up the help text which seemed suboptimal when there are only a small group of args specific to the function. I am still open to full on sub parser though so I'll leave this comment open until you weigh in further. (I have a pic in the PR conversation of the grouping)